english/coffee/hollywood/hollywood.md

```diff
@@ -1,3 +1,5 @@
 After writing, what you believe to be to be, a best-selling novel you decided to try to market the book right
-and head to Hollywood. The only choice now is how to get there. You’re afraid of flying and don’t care much for 
-public transportation. 
\ No newline at end of file
+and head to Hollywood. The only choice now is how to get there. You’re afraid of flying and don’t care much for
+public transportation.
+
+Obviously, you will have to travel old-fashioned Hollywood loner style. As a hobo, hopping cargo trains. You pack your manuscript into a trusty bag, pull a cap down over your head and head to the tracks. It seems like a train will never come, but when one does, you run along side and leap up to grab the rail.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.